app/javascript/components/upload/ProcessedExpressionStep.jsx

```diff
@@ -58,12 +60,13 @@ function ProcessedUploadForm({
     { !isEnabled &&
       <div className="row">
         <div className="col-md-12 padded">
-          Uploading a raw count matrix is now required in order to access to processed matrix uploads.
-          <br/>
-          <br/>
-          If you are unable or do not wish to upload a raw count matrix, you can request an exemption using the link below.
-          <br/>
-          <br/>
+          <p className="left-margin">
+            Uploading a raw count matrix is now required in order to access to processed matrix uploads.
```

Comment:
```suggestion
            Uploading a raw count matrix and associated barcodes/features files  is required before uploading a processed matrix.
```

Reply:
Since they can add a raw counts matrix as dense or sparse, I might edit this message to say:

`Uploading a raw count matrix is required before uploading a processed matrix.  Raw count matrices in sparse (MTX) format must also include associated features/barcodes files.`